### PR TITLE
[release] v0.6.0

### PR DESCRIPTION
**Pull request by yuanming-hu**
Related issue = #554 

I tested the release mode of v0.5.15 (pip-installed), and the OpenGL backend works well. I think we are finally safe to release v0.6. The link does not work yet, since I can only create the page after this PR is merged. 

[[Click here for the format server]](http://kun.csail.mit.edu:31415/)

### CODE REVIEW

**Comment by yuanming-hu** [comment]
@archibate Could you take care of this release? Tonight I have to rush for a presentation tomorrow. Let's welcome the world to try your OpenGL backend :-) You know how to trigger Linux build after emerging this PR.

File: README.md [hunk 1]
```diff
@@ -22,8 +23,11 @@ Advanced features of Taichi include [spatially sparse computing](https://taichi.
 ## Installation [![Downloads](https://pepy.tech/badge/taichi-nightly)](https://pepy.tech/project/taichi-nightly)
 
 ```bash
-python3 -m pip install taichi       # Latest version: 0.5.15
+python3 -m pip install taichi       # Latest version: 0.6.0
 ```
+
+**News: (Apr 29, 2020) The OpenGL Compute Shader backend (by [彭于斌 (archibate)](https://github.com/archibate)** is officially released in [**v0.6.0**](https://github.com/taichi-dev/taichi/releases/tag/v0.6.0)! Please try it out with `ti.init(arch=ti.opengl)`.
```
Comment (archibate) [inline README.md:29]:
```suggestion
**News: (Apr 29, 2020) The OpenGL Compute Shader backend by [彭于斌 (archibate)](https://github.com/archibate)** is officially released in [**v0.6.0**](https://github.com/taichi-dev/taichi/releases/tag/v0.6.0)! Please try it out with `ti.init(arch=ti.opengl)`.
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <div align="center">
   <img width="500px" src="https://github.com/yuanming-hu/taichi/raw/master/misc/logo.png">
-  <h3> <a href="https://taichi.readthedocs.io/en/latest/"> Documentation </a> | <a href="https://taichi.readthedocs.io/en/latest/hello.html"> Tutorial </a> | <a href="https://github.com/yuanming-hu/taichi/tree/master/examples"> Examples </a> | <a href="https://taichi.readthedocs.io/en/latest/contributor_guide.html"> Contributor Guidelines </a> | <a href="https://forum.taichi.graphics/"> Forum </a> </h3>
+  <h3> <a href="https://taichi.readthedocs.io/en/latest/hello.html"> Tutorial </a> | <a href="https://github.com/yuanming-hu/taichi/tree/master/examples"> Examples </a> | <a href="https://taichi.readthedocs.io/en/latest/contributor_guide.html"> Contributor Guidelines </a> | <a href="https://forum.taichi.graphics/"> Forum </a> </h3>
+  <h3> <a href="https://taichi.readthedocs.io/en/latest/"> Documentation </a> |<a href="https://taichi.readthedocs.io/zh_CN/latest/"> 简体中文文档 </a> </h3>
```
Comment (archibate) [inline README.md:4]:
```suggestion
  <h3> <a href="https://taichi.readthedocs.io/en/stable/"> Documentation </a> |<a href="https://taichi.readthedocs.io/zh_CN/latest/"> 简体中文文档 </a> </h3>
```

**Comment by k-ye** [comment]
@archibate  Why are you not mentioning your OpenGL backend in https://github.com/taichi-dev/taichi/releases/tag/v0.6.0....? Isn't that the single biggest thing to highlight in  0.6.0 ...? Congratulations!

**Pull request merged**